mock/docs/site-defaults.cfg

```diff
@@ -186,6 +186,18 @@
 #config_opts['bootstrap_chroot_additional_packages'] = []
 #config_opts['bootstrap_module_setup_commands'] = []
 
+# The following buildroot-image options are internal to Mock (at least for
+# now).  Please use the --buildroot-image option instead, and avoid setting
+# these in Mock configuration files.  The semantics are similar to the
+# s/buildroot/bootstrap/ options mentioned earlier.
+#config_opts['use_buildroot_image'] = False
+#config_opts['buildroot_image'] = None
+#config_opts['buildroot_image_skip_pull'] = False
+#config_opts['buildroot_image_ready'] = False
+#config_opts['buildroot_image_fallback'] = True
+#config_opts['buildroot_image_keep_getting'] = 120
+#config_opts['buildroot_image_assert_digest'] = None
```

Comment:
I think at least last 5 options deserve description what they should do and what is the meaning of the values.

Reply:
I'm afraid I'm going to copy-paste text that is used for the `s/buildroot/bootstrap` options, that's why I mentioned `The semantics are similar to the ...` part, but :shrug:

Reply:
I changed the header to this to make it more obvious:
```
+# These are used by --buildroot-image option logic.
+# The semantics are similar to the *bootstrap* counterparts above, e.g., the
+# option 'buildroot_image' is similar to 'bootstrap_image'.
```

Marking resolved, but feel free to unresolve this thread if you think we deserve more tweaks.